docs/sources/basics/exemplars/view-exemplars.md

```diff
@@ -0,0 +1,63 @@
+weight = 400
++++
+
+# View exemplar data
+
+When support for exemplar support is enabled for a Prometheus data source, you can view exemplar data either in the Explore view or from the Loki log details. 
+
+## In Explore
+
+Explore visualizes exemplar traces as highlighted stars alongside metrics data. For more information on how Explore visualizes trace data, refer to [tracing in Explore]({{< relref "../../explore/trace-integration.md" >}}).
+
+To examine the details of an exemplar trace:
+
+1. Place your cursor over an exemplar (highlighted star). Depending on your backend trace data source, you will see a blue button with the label `Query with <data source name>`. In the following example, the tracing data source is Tempo. 
+
+{{< docs-imagebox img="/img/docs/basics/exemplar-explore-details.png" class="docs-image--no-shadow" max-width= "750px" caption="Screenshot showing Exemplar details" >}}
+
+1. Click the **Query with Tempo** option next to the `traceID` property. The trace details, including the spans within the trace are listed in a separate panel on the right.
+
+{{< docs-imagebox img="/img/docs/basics/exemplar-explore-view.png" class="docs-image--no-shadow" max-width= "750px" caption="Explorer view with panel showing trace details" >}}
+
+For more information on how to drill down and analyse the trace and span details, refer to the [Analyze trace and span details](#analyze-trace-and-spans) section.
+
+
+## In logs
+
+You can also view exemplar trace details from the Loki logs. Use regex within the Derived fields links for Loki to extract the `traceID` information. Now when you expand Loki logs, you can see a `traceID` property under the **Detected fields** section. To learn more about how to extract a part of a log message into an internal or external link, refer to [using derived fields in Loki]({{< relref "../../explore/logs-integration.md" >}}).
+
+To view the details of an exemplar trace:
+
+1. Expand a log line and scroll down to the `Detected fields` section. Depending on your backend trace data source, you will see a blue button with the label `<data source name>`. In the following example, the tracing data source is Tempo.
+
+{{< docs-imagebox img="/img/docs/basics/exemplar-loki-logs.png" class="docs-image--no-shadow" max-width= "750px" caption="Loki logs and trace details panel" >}}
+
+1. Click the **Tempo** button next to the `traceID` property. The details, including the spans within the trace, are listed in a separate panel.
+
+For more information on how to drill down and analyse the trace and span details, refer to the [Analyze trace and span details](#analyze-trace-and-spans) section.
+ 
+ ## Analyze trace and spans
+
+ This panel shows the details of the trace in different segments. 
+ 
+ - The top segment shows the Trace ID to indicate that the query results correspond to the specific trace. 
+
+    You can add more traces to the results using the `Add Query` button.
```

Comment:
Why is query capitalized here, I am guessing you're matching the UI? Any chance of getting it switched to lowercase if it is like this in the UI?

Reply:
fixed.